tests/06-enums.rs

```diff
@@ -39,21 +36,30 @@
 //         Thirteen = 0b1101,
 //     }
 //
-//     ...
+// then the number of bits required to represent the struct is coerced to N.
+// Calling a getter on the struct returns a Result<T, EnumError>, where an
+// EnumError has an UnknownValue variant that can provide the raw value.
+//
 //     let mut bitfield = MyBitfield::new();
 //     assert_eq!(0, bitfield.small_prime().unwrap_err().raw_value());
 //
 //     bitfield.set_small_prime(SmallPrime::Seven);
 //     let p = bitfield.small_prime().unwrap_or(SmallPrime::Two);
+//
+// The getter for an enum marked with the bits attribute will return a Result
+// even if the struct has a number of variants which is a power of two.
```

Comment:
I think this is outdated now. ;)

Reply:
The example code was quite a bit outdated too, so updated that.